### PR TITLE
Add Order Button for GET and OpenTable

### PR DESCRIPTION
**Pull request by whoiswillma**
## Overview

Add order button (OpenTable for dining halls, GET for cafe's). Waiting on backend to implement the URLs for OpenTable and GET.

## Changes Made

  - Keep track of a bunch of scroll view properties
  - Parallax effect when scrolling the button
  - Add open URL for GET

## Test Coverage

Manual testing on a physical iPhone 11

## Screenshots

<details>

  <summary>OpenTable Button</summary>

  
  
IMG_2651

IMG_2650

</details>

<details>

  <summary>GET Button</summary>

  

IMG_2650

</details>

### CODE REVIEW

File: Eatery/Controllers/Eateries/Campus/CampusMenuViewController.swift [hunk 1]
```diff
@@ -123,10 +156,168 @@ class CampusMenuViewController: EateriesMenuViewController {
...
+        }
+
+        unhighlighOrderButton()
+    }
+
+    @objc private func highlightOrderButton() {
+        orderButton.backgroundColor = .darkEateryBlue
+    }
+
+    @objc private func unhighlighOrderButton() {
```
Comment (ywsang) [inline Eatery/Controllers/Eateries/Campus/CampusMenuViewController.swift]: spelling typo?

**Review by ywsang — APPROVED** [review]
🔥 some very interesting stuff with the scrolling/animation!

File: Eatery/Controllers/Eateries/Campus/CampusMenuViewController.swift [hunk 2]
```diff
@@ -123,10 +156,180 @@ class CampusMenuViewController: EateriesMenuViewController {
         }
     }
 
+    private func setUpOrderButton() {
+        orderButton.setImage(UIImage(named: "link"), for: .normal)
+
+        switch eatery.reservationType {
+        case .get:
+            if let url = URL.getUrl, UIApplication.shared.canOpenURL(url) {
```
Comment (spd75) [inline Eatery/Controllers/Eateries/Campus/CampusMenuViewController.swift:164]: What is the purpose of having this if let for GET but not for OpenTable?

**Review by spd75 — COMMENTED** [review]
All looks good but check the if let

**Pull request merged**